### PR TITLE
Additional fixes to the docker documentation

### PR DESCRIPTION
**Pull request by cybsecmaster**

### CODE REVIEW

**Comment by cybsecmaster** [comment]
Ticket DOC-37

File: docs/thehive/installation/docker/docker-compose.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+          cluster-name = thehive
+          keyspace = thehive
+          }
+      }
+      index.search {
+          backend = elasticsearch
+          hostname = ["elasticsearch"]
+          index-name = thehive
+      }
+    }
```
Comment (To-om) [inline docs/thehive/installation/docker/docker-compose.md:59]: fix indentation
```suggestion
    db.janusgraph {
      storage {
        backend = cql
        hostname = ["cassandra"]
        cql {
          cluster-name = thehive
          keyspace = thehive
        }
      }
      index.search {
        backend = elasticsearch
        hostname = ["elasticsearch"]
        index-name = thehive
      }
    }
```

File: docs/thehive/installation/docker/docker-compose.md [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+        <!--<logger name="org.janusgraph.diskstorage.es.rest.RestElasticSearchClient" level="TRACE"/>-->
+        <logger name="org.thp.scalligraph.models" level="INFO"/>
+        <logger name="org.thp" level="INFO"/>
+
+        <root level="INFO">
+            <appender-ref ref="ASYNCFILE"/>
+            <appender-ref ref="ASYNCSTDOUT"/>
+        </root>
+
+    </configuration>
```
Comment (To-om) [inline docs/thehive/installation/docker/docker-compose.md:155]: This logback.xml is the production one (not dev):
```suggestion
    <?xml version="1.0" encoding="UTF-8"?>
    <configuration debug="false">

        <conversionRule conversionWord="coloredLevel"
                        converterClass="play.api.libs.logback.ColoredLevel"/>

        <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
            <file>/var/log/thehive/application.log</file>
            <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
                <fileNamePattern>/var/log/thehive/application.%i.log.zip</fileNamePattern>
                <minIndex>1</minIndex>
                <maxIndex>10</maxIndex>
            </rollingPolicy>
            <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
                <maxFileSize>10MB</maxFileSize>
            </triggeringPolicy>
            <encoder>
                <pattern>%date [%level] from %logger in %thread %replace(\(%X{userId}@%X{organisation}\) ){'\(@\) ',''}[%X{kamonTraceId}] %message%n%xException</pattern>
            </encoder>
        </appender>

        <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
            <encoder>
                <pattern>%coloredLevel %logger{15} %replace(\(%X{userId}@%X{organisation}\) ){'\(@\) ',''}[%X{kamonTraceId}] %message%n%xException{10}
                </pattern>
            </encoder>
        </appender>

        <appender name="ASYNCFILE" class="ch.qos.logback.classic.AsyncAppender">
            <appender-ref ref="FILE"/>
        </appender>

        <appender name="ASYNCSTDOUT" class="ch.qos.logback.classic.AsyncAppender">
            <appender-ref ref="STDOUT"/>
        </appender>

        <!-- do not set the following logger to TRACE -->
        <logger name="org.thp.scalligraph.traversal" level="INFO"/>
        <logger name="org.reflections8.Reflections" level="ERROR" />
        <logger name="org.janusgraph.graphdb.database.management.ManagementLogger" level="OFF"/>
        <logger name="org.janusgraph.graphdb.database.IndexSerializer" level="ERROR"/>
        <logger name="org.apache.kafka" level="WARN" />
        <logger name="org.docx4j" level="WARN"/>
        <logger name="org.docx4j.model.PropertyResolver" level="OFF" />

        <logger name="org.thp" level="INFO"/>

        <root level="INFO">
            <appender-ref ref="ASYNCFILE"/>
            <appender-ref ref="ASYNCSTDOUT"/>
        </root>

    </configuration>
```

File: docs/thehive/installation/docker/docker-compose.md [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+    ```
+
+    &nbsp;
+
+    **`secret.conf`**: Use the following commands to create the file:
+
+    ```bash
+    key=$(dd if=/dev/urandom bs=1024 count=1 | tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1)
+    echo "play.http.secret.key=\"$key\"" | sudo tee -a ~/thehive/thehive/config/secret.conf
+    ```
```
Comment (To-om) [inline docs/thehive/installation/docker/docker-compose.md:165]: Root permission is not needed here:
```suggestion
    ```bash
    key=$(dd if=/dev/urandom bs=1024 count=1 | tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1)
    echo "play.http.secret.key=\"$key\"" > ~/thehive/thehive/config/secret.conf
    ```
```

**Review by To-om — CHANGES_REQUESTED** [review]

**Comment by nadouani** [comment]
@cybsecmaster please address the comments

**Pull request closed without merging**